cfgov/unprocessed/css/apps/owning-a-home/form-explainer.less

```diff
@@ -0,0 +1,447 @@
+            margin-bottom: unit( ( @grid_gutter-width - 4 ) / @base-font-size-px, em );
+            height: 1px;
+        }
+
+        .o-expandable_content:after {
+            margin: 0;
+        }
+
+        .o-expandable_category {
+            .cf-icon();
+        }
+    }
+
+    .o-expandable-group .o-expandable__form-explainer-checklist {
+        &,
+        &:first-child,
+        &.o-expandable__collapsing {
+            background: @gold-20;
+            border: 3px solid;
+            border-color: @gold-20;
+            transform: translateX( 0 );
+        }
+        &:hover,
+        &.hover-has-attention {
+            border-color: @gold-80;
+        }
+        &.o-expandable__expanded,
+        &.o-expandable__expanding,
+        &.has-attention {
+            border-color: @gold-80;
+            .o-expandable_header {
+                background: @gold-80;
+            }
+            .o-expandable_category:before {
+                content: "\e101"; // .cf-icon-approved-round
+                color: @black;
+            }
+            .o-expandable_content:before {
+                background: @gold-20;
+            }
+        }
+        .o-expandable_category:before {
+            content: "\e101"; // .cf-icon-approved-round
+            color: @gold-80;
+        }
+        .o-expandable_content:before {
+            background: @gold-80;
+        }
+    }
+
+    .o-expandable-group .o-expandable__form-explainer-alerts {
+        &,
+        &:hover,
+        &.o-expandable__expanded,
+        &.o-expandable__expanding,
+        &.has-attention, {
+            background: @red-20;
+        }
+        .o-expandable_category:before {
+            content: "\e103"; // .cf-icon-error-round
+            color: @red-80;
+        }
+        .o-expandable_content:before {
+            background: @red-60;
+        }
+    }
+
+    .o-expandable-group .o-expandable__form-explainer-definitions {
+        &,
+        &:first-child
+        &.o-expandable__collapsing {
+            background: @teal-20;
+            border: 3px solid;
+            border-color: @teal-20;
+            transform: translateX( 0 );
+        }
+        &:hover,
+        &.hover-has-attention {
+            border-color: @teal-60;
+        }
+        &.o-expandable__expanded,
+        &.o-expandable__expanding,
+        &.has-attention {
+            & {
+                border-color: @teal-60;
+            }
+            .o-expandable_header {
+                background: @teal-60;
+            }
+            .o-expandable_category:before {
+                content: "\e105"; // .cf-icon-help-round
+                color: @black;
+            }
+            .o-expandable_content:before {
+                background: @teal-20;
+            }
+        }
+        .o-expandable_category:before {
+            content: "\e105"; // .cf-icon-help-round
+            color: @teal;
+        }
+        .o-expandable_content:before {
+                background: @teal-60;
```

Comment:
Minor, extra indentation here and below.

Reply:
Fixed.